fairscale/experimental/wgit/sha1_store.py

```diff
@@ -70,49 +91,20 @@ def add(self, file_path: str) -> None:
                 "time_stamp": change_time,
             }
             # Populate the meta_data file with the meta_data and git add
-            self._add_metadata_to_json(metadata)
+            self._write_to_json(metadata_file, metadata)
```

Comment:
I am curious why would sha1_store handle the metadata file? I mean it should indeed handle the ref count file, but the per-input-file metadata file should be handled by the repo class? Let's chat in more details during our VC.